src/constants.ts

```diff
@@ -3,4 +3,9 @@ export enum DispatchActions {
   ADD_PARTY_MEMBER = 'add_party_member',
   SET_TIP = 'set_tip',
   SET_TAX = 'set_tax',
+  RESET = 'reset'
 }
+
+export default { 
```

Comment:
not sure why but the lint was throwing an error for not having an export default so randomly added one